scene/3d/path.cpp

```diff
@@ -43,6 +43,15 @@ void Path::_curve_changed() {
 	if (is_inside_tree()) {
 		emit_signal("curve_changed");
 	}
+
+	// update the configuration warnings of all children of type OrientedPathFollows
+	if (is_inside_tree()) {
+		for (int i = 0; i < get_child_count(); i++) {
+			if (OrientedPathFollow *child = Object::cast_to<OrientedPathFollow>(get_child(i))) {
```

Comment:
We prefer to avoid assignments in conditionals in Godot's code base.

Reply:
thanks, fixed.